pyperformance/commands.py

```diff
@@ -191,8 +191,8 @@ def cmd_run(options, benchmarks):
 
     if errors:
         print("%s benchmarks failed:" % len(errors))
-        for name in errors:
-            print("- %s" % name)
+        for name, reason in errors:
+            print("- %s (%s)" % (name, reason))
```

Comment:
Should this be in a separate pull request?

Reply:
The timeout implementation introduces a new way that a benchmark can fail. This is really to differentiate the ways a benchmarks can terminate hence I thought to be explicit and communicate it back to the user what the reason of the termination is.
If you want I can split it in an extra PR.

Reply:
It's not up to me to decide that. I'm not a maintainer. :)

Reply:
This change to show the reason makes sense to me.